tests/robot-tests/run_tests.py

```diff
@@ -62,8 +62,15 @@
                     default="74.0.3729.6",
                     metavar="{version}",
                     help="specify which version of chromedriver to use")
+parser.add_argument("--admin-pass",
```

Comment:
What is the admin password actually being used for?  Is it a permissions thing in Robot, or a password for a DFE user etc?

Reply:
It's the azure/microsoft login password to access the admin app. Robot can get the email through an env variable set in the pipeline. But because the password is secret, azure will only allow you to pas it as an argument

Reply:
Cool - it might be worth then mentioning that somewhere in a comment for folks like me who wouldn't know its purpose at a glance?

Reply:
done